digdag-standards/src/main/java/io/digdag/standards/operator/aws/EmrOperatorFactory.java

```diff
@@ -181,10 +181,37 @@ public TaskResult runTask()
             AWSCredentials credentials = credentials(tag);
 
             SecretProvider awsSecrets = context.getSecrets().getSecrets("aws");
+            SecretProvider s3Secrets = awsSecrets.getSecrets("s3");
+            SecretProvider emrSecrets = awsSecrets.getSecrets("emr");
+            SecretProvider kmsSecrets = awsSecrets.getSecrets("kms");
+
+            Optional<String> emrEndpoint = Aws.first(
+                    () -> emrSecrets.getSecretOptional("endpoint"),
+                    () -> params.getOptional("endpoint", String.class));
```

Comment:
```
  () -> emrSecrets.getSecretOptional("endpoint"),
  () -> awsSecrets.getSecretOptional("emr.endpoint"),
```
Overall looks good, but these are the same one and `() -> awsSecrets.getSecretOptional("emr.endpoint")` should be removed.